### PR TITLE
Update sltsheryl AboutUs 

### PR DESCRIPTION
**Pull request by sltsheryl**
Handles #127

### CODE REVIEW

File: docs/team/sltsheryl.md [hunk 1]
```diff
@@ -4,37 +4,63 @@ title: Sheryl-Lynn Tan's Project Portfolio Page
...
 
 Given below are my contributions to the project.
 
-* **New Feature**: [To be added soon]
-    * What it does: [To be added soon]
-    * Justification: [To be added soon]
-    * Highlights: [To be added soon]
-    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+####New Enhancement: User Interface
+
```
Comment (carriezhengjr) [inline docs/team/sltsheryl.md:12]: Not sure if we need to follow the format given, i.e. What it does, Justification, Highlights, Credits.
Refer to https://se-education.org/addressbook-level3/team/johndoe.html for examples.

File: docs/team/sltsheryl.md [hunk 2]
```diff
@@ -4,37 +4,63 @@ title: Sheryl-Lynn Tan's Project Portfolio Page
...
+Their respective ```.java``` files were also refactored to make the application more dynamic to changes. 
+
+The ```DarkTheme.css``` file was extensively refactored to update the UI interface.
+
+####New Feature: Add Interest
+
+I also worked on the add interest feature. The ```add int``` command gives users a hassle-free way of adding new interests to a student.
 
-* **New Feature**: [To be added soon]
+```AddInterestCommandTest``` and ```AddInterestParserCommandTest``` were also created to include testing coverage for ```AddInterestCommand``` and ```AddInterestParserCommand``` classes.
```
Comment (carriezhengjr) [inline docs/team/sltsheryl.md:37]: Perhaps the format could be more consistent, i.e. 
__Changes to Files Made:__ `AddInterestCommandTest` and ...

File: docs/team/sltsheryl.md [hunk 3]
```diff
@@ -4,37 +4,63 @@ title: Sheryl-Lynn Tan's Project Portfolio Page
...
+```AddInterestCommandTest``` and ```AddInterestParserCommandTest``` were also created to include testing coverage for ```AddInterestCommand``` and ```AddInterestParserCommand``` classes. 
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=sltsheryl&breakdown=true)
+####Other Modifications: Align Person with Application Purpose
+Modifications to the ```Person``` fields were made so it is tailored to the function of Mass Linkers, such as the removal of ```Address``` and addition of new fields like ```Telegram```and ```GitHub```. 
+####Code contributed
 
-* **Project management**:
-    * [To be added soon]
+[RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=sltsheryl&breakdown=true)
```
Comment (carriezhengjr) [inline docs/team/sltsheryl.md:43]: This part didn't display correctly on my screen when I use the "Display  the rich diff" function at the top right corner under "Files changed" on GitHub. Not sure if the display is actually correct. Maybe you could double check the display for this part 🤔 :

####Other Modifications: Align Person with Application Purpose Modifications to the Person fields were made so it is tailored to the function of Mass Linkers, such as the removal of Address and addition of new fields like Telegramand GitHub. ####Code contributed

[RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=sltsheryl&breakdown=true)

**Review by carriezhengjr — COMMENTED** [review]
Content is good and detailed. 👍 
Some suggestions you could consider and displays that you could double check as they aren't displayed correctly on my end on GitHub.

File: docs/team/sltsheryl.md (hunk 1 above)
Comment (sltsheryl) [reply docs/team/sltsheryl.md]: Edited. 
Kept the New Feature as a header and not a bullet, cuz it's neater this way

File: docs/team/sltsheryl.md (hunk 2 above)
Comment (sltsheryl) [reply docs/team/sltsheryl.md]: Edited

File: docs/team/sltsheryl.md (hunk 3 above)
Comment (sltsheryl) [reply docs/team/sltsheryl.md]: Okay I added space, that's the issue

**Review by carriezhengjr — APPROVED** [review]
LGTM 👍

**Comment by sltsheryl** [comment]
> LGTM 👍

Not yet. Need to change Person to Student

**Pull request merged**